src/component/Options.ts

```diff
@@ -37,4 +37,6 @@ export enum FitType {
   Horizontal,
   /** Fit only vertically. */
   Vertical,
+  /** Fit and center the graph. */
```

Comment:
IMHO, we should avoid using graph as this is something related to mxgraph and is not related to the BPMN domain.

```suggestion
  /** Fit and center the BPMN Diagram. */
```